en/docs/apis/api-concepts.md

```diff
@@ -22,30 +22,29 @@ The access token is a string that the consumer passes in an HTTP header of the A
 
 ## Rate limiting
 Rate limiting limits the number of permitted requests to an API within a specified timeframe. You can use a rate-limiting strategy for the following use cases:
-    - Protect your APIs from common types of security attacks, such as certain types of Denial of Service (DoS) attacks 
-    - Regulate traffic according 
+
+- Protect your APIs from common types of security attacks, such as certain types of Denial of Service (DoS) attacks
+- Regulate traffic according to the infrastructure availability
```

Comment:
@shaniR Can you pls double check this? Whether this is the correct fix for missing part.

Reply:
Yes this is  the missing part.